Introduction of Skia Vulkan backend breaks build in  Ubuntu 22.04.4 LTS/Jammy (on WSL2)

With https://github.com/LadybirdBrowser/ladybird/commit/e713de115c394480b7d57653fe7b472db0dc7ce3, the compilation of DisplayListPlayerSkia.cpp fails with the errors;

```
In file included from /src/ladybird/Userland/Libraries/LibWeb/Painting/DisplayListPlayerSkia.cpp:31:
In file included from /src/ladybird/Build/ladybird/vcpkg_installed/x64-linux/include/skia/gpu/vk/GrVkBackendContext.h:12:
In file included from /src/ladybird/Build/ladybird/vcpkg_installed/x64-linux/include/skia/include/gpu/vk/GrVkTypes.h:13:
/src/ladybird/Build/ladybird/vcpkg_installed/x64-linux/include/skia/include/gpu/vk/VulkanTypes.h:107:56: error: use of undeclared identifier 'VkDeviceFaultAddressInfoEXT'
  107 |                                      const std::vector<VkDeviceFaultAddressInfoEXT>& addressInfos,
      |                                                        ^
/src/ladybird/Build/ladybird/vcpkg_installed/x64-linux/include/skia/include/gpu/vk/VulkanTypes.h:108:56: error: use of undeclared identifier 'VkDeviceFaultVendorInfoEXT'
  108 |                                      const std::vector<VkDeviceFaultVendorInfoEXT>& vendorInfos,
      |                                                        ^
```

Evaluating with -E, I see vulkan_core.h get pulled in by LibCore/VulkanContext.h, and it seems to grab the system installed version in /usr/include/vulkan -- This is from the system wide package libvulkan-dev @1.3.204.1-2 -- and does not include the VkDeviceFaultAddressInfoEXT definition.

So, we probably don't want to mix and match system vulkan and skia vulkan, it seems like ideally they'd both use the same version.

--- a/.github/actions/setup/action.yml
+++ b/.github/actions/setup/action.yml
@@ -25,7 +25,9 @@ runs:
         sudo add-apt-repository ppa:ubuntu-toolchain-r/test
 
         sudo apt-get update
-        sudo apt-get install autoconf autoconf-archive automake build-essential cmake libavcodec-dev fonts-liberation2 zip curl tar ccache clang-18 clang++-18 lld-18 gcc-13 g++-13 libstdc++-13-dev ninja-build unzip qt6-base-dev qt6-tools-dev-tools libqt6svg6-dev qt6-multimedia-dev libgl1-mesa-dev libpulse-dev libssl-dev libegl1-mesa-dev
+        sudo apt-get install autoconf autoconf-archive automake build-essential cmake libavcodec-dev fonts-liberation2 zip curl tar ccache clang-18 clang++-18 lld-18 gcc-13 g++-13 libstdc++-13-dev \
+            ninja-build unzip qt6-base-dev qt6-tools-dev-tools libqt6svg6-dev qt6-multimedia-dev libgl1-mesa-dev libpulse-dev libssl-dev libegl1-mesa-dev \
+            libx11-dev libxrandr-dev
 
         sudo update-alternatives --install /usr/bin/clang clang /usr/bin/clang-18 100
         sudo update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-18 100

--- a/Meta/CMake/vulkan.cmake
+++ b/Meta/CMake/vulkan.cmake
@@ -1,6 +1,7 @@
 if (NOT APPLE)
+    find_package(VulkanHeaders CONFIG QUIET)
     find_package(Vulkan QUIET)
-    if (Vulkan_FOUND)
+    if (VulkanHeaders_FOUND AND Vulkan_FOUND)
         set(HAS_VULKAN ON CACHE BOOL "" FORCE)
         add_compile_definitions(USE_VULKAN=1)
     endif()

--- a/Userland/Libraries/LibCore/CMakeLists.txt
+++ b/Userland/Libraries/LibCore/CMakeLists.txt
@@ -82,7 +82,6 @@ else()
 endif()
 
 if (HAS_VULKAN)
-    include_directories(${Vulkan_INCLUDE_DIR})
     list(APPEND SOURCES VulkanContext.cpp)
 endif()
 
@@ -112,5 +111,5 @@ if (ANDROID)
 endif()
 
 if (HAS_VULKAN)
-    target_link_libraries(LibCore PUBLIC ${Vulkan_LIBRARIES})
+    target_link_libraries(LibCore PUBLIC Vulkan::Vulkan Vulkan::Headers)
 endif()

--- a/Userland/Libraries/LibUnicode/CMakeLists.txt
+++ b/Userland/Libraries/LibUnicode/CMakeLists.txt
@@ -26,5 +26,5 @@ set(GENERATED_SOURCES ${CURRENT_LIB_GENERATED})
 
 serenity_lib(LibUnicode unicode)
 
-find_package(ICU REQUIRED COMPONENTS data i18n uc)
+find_package(ICU 74 REQUIRED COMPONENTS data i18n uc)
 target_link_libraries(LibUnicode PRIVATE ICU::i18n ICU::uc ICU::data)

--- a/vcpkg.json
+++ b/vcpkg.json
@@ -32,7 +32,15 @@
       "platform": "android"
     },
     "sqlite3",
-    "woff2"
+    "woff2",
+    {
+      "name": "vulkan",
+      "platform": "!android"
+    },
+    {
+      "name": "vulkan-headers",
+      "platform": "!android"
+    }
   ],
   "overrides": [
     {

